conf/application.conf

```diff
@@ -194,6 +195,7 @@ urls {
   additionalDocumentsReferenceCodes = "https://www.gov.uk/guidance/data-element-23-documents-and-other-reference-codes-national-of-the-customs-declaration-service-cds#table-2-national-document-codes"
   additionalDocumentsUnitCodes = "https://www.gov.uk/government/publications/measurement-unit-codes-for-data-elements-23-and-44-of-the-customs-declaration-service"
   licensesForExportingGoods = "https://www.gov.uk/export-goods"
+  exportDeclarationGuidance = "/customs-declare-exports/complete-declaration"
```

Comment:
Added a better alternative above, but it seems that there is no such endpoint at the moment in this service.